index.html

```diff
@@ -281,5 +281,30 @@ <h3 class="titleA">About My self</h3>
                 </li>
               </ul>
             </section>
-  
-  
\ No newline at end of file
+  <!--Works Section contact page-->
+    <section class="page6">
+      <div class="bag">
+      <div class="contact_container">
+            <h2>Contact me</h2>
+            <p>If you have an application you are 
+              interested in developing, a feature that you 
+              need built or a project that needs coding.<br> I’d love 
+              to help with it!</p>
+      </div>
+
+
+          <form action="https://formspree.io/f/xpzgwrza" method="post">
+
+            <input type="User name" maxlength="30" required="" placeholder="Enter Full Name">
```

Comment:
- [ ] Please pay careful attention to the feedback given to you by reviewers as I noticed the `type` attribute for the name input is still set to an undefined value, as mentioned previously there are only a specific set of predefined values you can set for the `type` attribute and since you're accepting a text here kindly change the value to `text`.